lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Ansible, but is an independent component
+
+# This particular file snippet, and this file snippet only, is BSD licensed.
```

Comment:
I think there will be a problem with the license. IIRC modules in Ansible ought to be GPLv3+ in order to ship with Ansible. The other problem is that you add copyright to other people from the ACI team, and that code is GPLv3, so whatever copyright is held from there, cannot be turned into BSD.

Reply:
I looked at a bunch of module utilities and they all have the same stock license verbiage at the top. Each of them references BSD license and not GPLv3. 

I removed the others from the ACI team and left myself. That’ll come in a future push.

Reply:
Forget about my statement, the module_utils is BSD ! I was confusing the modules with module_utils. Nevermind.